plugins/stack-overflow/src/search/StackOverflowSearchResultListItem/StackOverflowSearchResultListItem.tsx

```diff
@@ -48,6 +48,10 @@ export const StackOverflowSearchResultListItem = (
   const analytics = useAnalytics();
 
   const handleClick = () => {
+    if (!result) {
+      return;
```

Comment:
shouldn't it return `null` like that if statement below?

Reply:
Not sure why I added an extra check there, will remove.